frontend/device/android/device.lua

```diff
@@ -324,10 +336,16 @@ end
 
 function Device:info()
     local is_eink, eink_platform = android.isEink()
+    local product_type = android.getPlatformName()
 
     local common_text = T(_("%1\n\nOS: Android %2, api %3\nBuild flavor: %4\n"),
         android.prop.product, getCodename(), Device.firmware_rev, android.prop.flavor)
 
+    local platform_text = ""
+    if product_type ~= "android" then
+        platform_text = T(_("\nDevice type: %1\n"), product_type)
```

Comment:
For translation it's friendlier to keep the newlines outside. (And I hardly think the theoretical overhead of string concat is worth worrying about. ;-) )